src/Controls/src/Core/HandlerImpl/VisualElement.Impl.cs

```diff
@@ -36,6 +33,12 @@ protected override void OnPropertyChanged([CallerMemberName] string propertyName
 
 		public bool IsArrangeValid { get; protected set; }
 
+		public CornerRadius CornerRadius { get; set; }
+
+		public Color BorderColor { get; set; }
```

Comment:
The naming should be different?. For example: `Stroke` and `StrokeThickness` like in Shapes?